splice_ck/src/main/java/com/splicemachine/ck/command/RGetCommand.java

```diff
@@ -0,0 +1,81 @@
+/*
```

Comment:
If a table exists in multiple schemas, this will always select the latest one. It'd be nice to be able to specify the schema. We'd need to find a way to distinguish between the schema in which the table resides and the schema you already use here (boolean, int, etc.)
Also, table names and schema names should be normalised (`EngineUtils.validateTable`, `EngineUtils.validateSchema`) so that the end user can write `t2` instead of `T2` for example.

Reply:
Good point(s) :)